web/src/main/java/org/akaza/openclinica/control/managestudy/ViewStudySubjectServlet.java

```diff
@@ -76,6 +77,11 @@
     // request attribute for a discrepancy note
     public final static String ENROLLMENT_NOTE = "enrollmentNote";
     public final static String COMMON = "common";
+    public final static String OPEN_BRACKET = "[";
+    public final static String CLOSE_BRACKET = "]";
+    public final static String DOT_ESCAPED = "\\.";
+
+    public final static String visitBasedEventItempath=CoreResources.getField("visitBasedEventItem");
```

Comment:
can you commit this property in the datainfo.properties file with some example value for documentation purposes?

Reply:
I already added  the property sample in the story in the comment https://jira.openclinica.com/browse/OC-11318. Also added some instructions on how to test some criteria.
I don't want to check in my datainfo.properties file to master.

Reply:
sounds good, thanks